src/features/landing/components/Possibilities/Text/Text.tsx

```diff
@@ -0,0 +1,45 @@
+'use client';
+
+import { Button, Stack, Typography } from '@mui/material';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+function Text() {
```

Comment:
for what this component? first what i think when see the name of this component that he must show some text and that's all, need add more specific name